composer.json

```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^8.0",
```

Comment:
Why not `"php": "^7.3 || ^8.0"`?

Look at https://github.com/avtocod/specs-php/pull/9